stl/inc/flat_set

```diff
@@ -453,7 +454,7 @@ private:
             if (_Where == _End || !_Keys_equal(_Val, *_Where)) {
                 return _Cont.insert(_Where, _STD forward<_Ty>(_Val));
             }
-            return _Where;
+            return _Cont.begin() + (_Where - _Begin);

```

Comment:
I'm just making this line work (`const_iterator` to `iterator`); may need further improvements in the future.